src/pages/RepoPage/CoverageOnboarding/ActivationBanner/ActivationBanner.tsx

```diff
@@ -21,16 +23,21 @@ function ActivationBanner() {
     isBasicPlan(planData?.plan?.value) &&
     planData?.hasPrivateRepos &&
     isNewTrial
+  const seatsLimitReached = !planData?.plan?.hasSeatsLeft
 
-  if (!isTrialEligible) {
-    return null
+  if (isTrialEligible) {
+    return <TrialEligibleBanner />
```

Comment:
Do we want to maintain the "mt-4" className?

Reply:
Nope i handled it inside the component itself